### PR TITLE
892709: CRUD support in diagram control

### PR DESCRIPTION
**Pull request by BalaVigneshRaviChandran**

### CODE REVIEW

File: UG-Samples/DataBinding/DiagramWithCustomAdaptor/DiagramWithCustomAdaptor/Components/Pages/CustomAdaptorSample.razor [hunk 1]
```diff
@@ -0,0 +1,214 @@
+@page "/"
+@page "/CRUD/LocalData/DiagramWithCustomAdaptor"
```
Comment (SuganthiK963) [inline UG-Samples/DataBinding/DiagramWithCustomAdaptor/DiagramWithCustomAdaptor/Components/Pages/CustomAdaptorSample.razor:2]: remove multiple navigation links

File: UG-Samples/DataBinding/DiagramWithCustomAdaptor/DiagramWithCustomAdaptor/Components/App.razor [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <base href="/" />
+    <link rel="stylesheet" href="bootstrap/bootstrap.min.css" />
+    <link rel="stylesheet" href="app.css" />
+    <link rel="stylesheet" href="DiagramWithCustomAdaptor.styles.css" />
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <HeadOutlet @rendermode="InteractiveServer" />
+    <link href="https://cdn.syncfusion.com/blazor/26.1.35/styles/bootstrap5.css" rel="stylesheet" />
```
Comment (SuganthiK963) [inline UG-Samples/DataBinding/DiagramWithCustomAdaptor/DiagramWithCustomAdaptor/Components/App.razor:13]: why you refer the cdn link?

File: UG-Samples/DataBinding/DiagramWithCustomAdaptor/DiagramWithCustomAdaptor/Components/App.razor
Comment (SuganthiK963) [inline UG-Samples/DataBinding/DiagramWithCustomAdaptor/DiagramWithCustomAdaptor/Components/App.razor:1]: Add Readme file for all the samples.

Comment (SuganthiK963) [inline UG-Samples/DataBinding/DiagramWithCustomAdaptor/DiagramWithCustomAdaptor/Components/App.razor:1]: Add .Net6 and .Net8 version csproj for all the samples

File: UG-Samples/DataBinding/DiagramWithCustomAdaptor/DiagramWithCustomAdaptor/Components/Pages/Counter.razor [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿@page "/counter"
```
Comment (SuganthiK963) [inline UG-Samples/DataBinding/DiagramWithCustomAdaptor/DiagramWithCustomAdaptor/Components/Pages/Counter.razor:1]: Remove unwanted files

File: UG-Samples/DataBinding/DiagramWithCustomAdaptor/DiagramWithCustomAdaptor/Components/Pages/Grid.razor [hunk 4]
```diff
@@ -0,0 +1,134 @@
+﻿@page "/Grid"
+@rendermode InteractiveServer
+@using Syncfusion.Blazor
+@using Syncfusion.Blazor.Data
+@using Syncfusion.Blazor.Grids
+@using TextAlign = Syncfusion.Blazor.Grids.TextAlign
+
+<SfGrid TValue="Order" ID="Grid" AllowSorting="true" AllowFiltering="true" AllowPaging="true" Toolbar="@(new List<string>() { "Add", "Delete", "Update", "Cancel" })">
```
Comment (SuganthiK963) [inline UG-Samples/DataBinding/DiagramWithCustomAdaptor/DiagramWithCustomAdaptor/Components/Pages/Grid.razor:8]: Why you add the Grid sample?

**Review by Keerthivasan-Ramamoorthy — APPROVED** [review]

**Pull request merged**